game/dcs/aircrafttype.py

```diff
@@ -400,6 +400,24 @@ def _set_props_overrides(
             for k in config:
                 if k in aircraft.property_defaults:
                     aircraft.property_defaults[k] = config[k]
+                    # In addition to setting the property_defaults, we have to set the "default" property in the
+                    # value of aircraft.properties for the key, as this is used in parts of the codebase to get
+                    # the default value. We have to instantiate a new UnitPropertyDescription object as this dataclass
+                    # is set to frozen in pydcs.
+                    aircraft.properties[k] = UnitPropertyDescription(
```

Comment:
https://docs.python.org/3/library/dataclasses.html#dataclasses.replace so we don't have to keep this up-to-date with changes to `UnitPropertyDescription`.

Reply:
Thanks, didn't know that existed :)